.github/workflows/test-regression.yaml

```diff
@@ -91,6 +91,7 @@ jobs:
 
       - name: Report build
         run: |
+          # TODO: create script to create build issue markdown and create issue
```

Comment:
Are you talking about build failures or the pending state?

Reply:
Maybe something like:
```
# TODO: Create placeholder issue on pending
# TODO: Update issue on build success
# TODO: Update issue on build failure
```

Reply:
I believe it should create an issue when pending and update the issue upon build success or build failure. I think I'll need a script to generate the markdown